src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -1146,6 +1146,7 @@ internal void RemoveChild(View child)
             // If the view had focus, it clears focus.
             if (child == FocusManager.Instance.GetCurrentFocusView())
             {
+                Tizen.Log.Debug("NUI", $"ClearFocus id:({child.ID})\n");
```

Comment:
```suggestion
                Tizen.Log.Debug("NUI", $"ClearFocus due to View id:({child.ID}) removed from scene.\n");
```

Could you write some more informations at this line? Currently, we don't know whether app call ClearFocus itself, or not.